template/analytics.cve.warning/body.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
```

Comment:
The folder name should be 'analytics.notify.cve' and not 'analytics.cve.warning'.  This folder name should be same as notification type.  Check notification payload for reference.